### ISSUE TITLE
Using mouse clicks for hotkeys

### ISSUE THREAD

**Issue by jotumn** [SethRobinson/UGT#27]
If you play with mouse and keyboard, using something like ctrl+right click (or just additional mouse keys) would be easier to use.

**Comment by SethRobinson** [comment]
I agree.  This is kind of a hassle to do though,  we'd need to install a low level mouse hook via a dll (see SetWindowsHookEx)  to detect the clicks while we're not in focus.

**Comment by storycoolbro** [comment]
if you've still not thought of this in the last 2 years. just get a mouse with programmable extra buttons and set one of them as a multi key and set it to w/e your shortcut is, easiest way to do this just have to buy a mouse with programmable buttons if you don't already have one, and they're not that expensive nowadays. been using my extra mouse buttons for UGT since I set it up.
